package.json

```diff
@@ -5,6 +5,13 @@
   "main": "dist/dedent.js",
   "types": "./dist/dedent.d.ts",
   "module": "./dist/dedent.mjs",
+  "exports": {
+    ".": {
+      "types": "./dist/dedent.d.ts",
+      "import": "./dist/dedent.mjs",
+      "default": "./dist/dedent.js"
+    }
+  },
```

Comment:
Just to be safe 🙂 - @otakustay does this look right to you?

I'll note that as of #51, the types are now in `./dist/dedent.d.ts`.